mb-sdk/src/data/store.rs

```diff
@@ -245,6 +228,28 @@ impl From<TokenMetadata> for TokenMetadataCompliant {
     }
 }
 
+/// Metadata and meta-metadata for tokens minted on store v2
+#[derive(Clone, BorshDeserialize, BorshSerialize)]
+pub struct MintingMetadata {
+    /// Number of tokens minted on this metadata
+    pub minted: u32,
+    /// Number of tokens minted on this metadata
+    pub burned: u32,
+    /// Price required to mint on this metadata
+    pub price: near_sdk::Balance,
+    /// Maximum amount of tokens allowed to be minted, no restrictions if `None`
+    pub minting_cap: Option<u32>,
```

Comment:
ubernit: I would call this just `mint_cap` because accessing this field would look a bit prettier.

`minting_metadata.minting_cap` --> `minting_metadata.mint_cap`

Reply:
Calling it `max_supply` now